### PR TITLE
Add required dependency for Github Actions Java ARM Test Suite.

### PR DESCRIPTION
**Pull request by celeste-zeng**
- Test failed due to missing dependency, added in this PR.
------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

**Comment by celeste-zeng** [comment]
R:@abacn

**Review by Abacn — APPROVED** [review]

**Comment by Abacn** [comment]
Run Java_Examples_Dataflow_ARM PostCommit

**Comment by Abacn** [comment]
Looks like phrase comment workflow has some bug:

```
Setup repository

Prepare all required actions
Getting action download info
Run ./.github/actions/setup-action
Run echo "The comment Run Java_Examples_Dataflow_ARM PostCommit does not match the phrase for this instance:  Run Java_Examples_Dataflow_ARM PostCommit [1](https://github.com/apache/beam/actions/runs/5862881738/job/15895405830#step:4:1)1. Exiting."
The comment Run Java_Examples_Dataflow_ARM PostCommit does not match the phrase for this instance:  Run Java_Examples_Dataflow_ARM PostCommit 11. Exiting.
Error: Process completed with exit code 1.
```
https://github.com/apache/beam/actions/runs/5862881738/job/15895405830

**Comment by Abacn** [comment]
Run Java_Examples_Dataflow_ARM PostCommit 8

**Comment by celeste-zeng** [comment]
Due to https://github.com/apache/beam/pull/27852, I think now all rerun requires a version specified.

**Comment by Abacn** [comment]
It's not classNotFoundException now, but `java.lang.IllegalArgumentException at WordCountIT.java:58`

File: .github/workflows/beam_PostCommit_Java_Examples_Dataflow_ARM.yml [hunk 1]
```diff
@@ -129,4 +129,6 @@ jobs:
         uses: actions/upload-artifact@v3
         with:
           name: java-code-coverage-report
-          path: "**/build/test-results/**/*.xml"
+          path: |
+              "**/build/test-results/**/*.xml"
+              "**/build/reports/tests/**/*.xml"
```
Comment (Abacn) [inline .github/workflows/beam_PostCommit_Java_Examples_Dataflow_ARM.yml:134]: upload all files in reports/tests  (html, css, etc)

File: .github/workflows/beam_PostCommit_Java_Examples_Dataflow_ARM.yml [hunk 2]
```diff
@@ -129,4 +129,6 @@ jobs:
         uses: actions/upload-artifact@v3
```
Comment (Abacn) [inline .github/workflows/beam_PostCommit_Java_Examples_Dataflow_ARM.yml:129]: a condition

`if: always()`

should make upload step alway run. ref: https://github.com/actions/upload-artifact/issues/328#issuecomment-1159824887

**Comment by Abacn** [comment]
https://github.com/apache/beam/blob/3530d44745a864970a265c7996069d7cde95e8bf/runners/google-cloud-dataflow-java/arm/build.gradle#L101

Change to:

`--workerMachineType`

**Comment by Abacn** [comment]
https://github.com/apache/beam/blob/3530d44745a864970a265c7996069d7cde95e8bf/runners/google-cloud-dataflow-java/arm/build.gradle#L89

java${javaVer}_sdk -> ${javaVer}_sdk

**Pull request merged**